pkg/database/realm.go

```diff
@@ -165,6 +165,11 @@ type Realm struct {
 	WelcomeMessage    string  `gorm:"-"`
 	WelcomeMessagePtr *string `gorm:"column:welcome_message; type:text;"`
 
+	// AgencyBackgroundColor and AgencyImage are synced from the Google
+	// ENX-Express sync source
+	AgencyBackgroundColor string `gorm:"column:agency_background_color; type:text; not null; default: '#ffffff'"`
```

Comment:
I think these should be nullable fields (see WelcomeMessage and WelcomeMessagePtr above)